iceoryx_posh/CMakeLists.txt

```diff
@@ -12,6 +12,38 @@ include(cmake/install_help_and_config.cmake)
 include(IceoryxPackageHelper)
 include(IceoryxPlatformDetection)
 
+# configure deployment
```

Comment:
Is it a good idea to define these configuration parameters into a separate file ?

Reply:
it could be moved into a seperate cmake file